donkeycarManagerFront/src/components/uiTraining.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <img classs="svgUI" ref="uiSvg" id="svg-ui">
+</template>
+<script>
+const imageCount = 4
+export default {
+  data () {
+    return {
+      loop: null
+    }
+  },
+  mounted () {
+    let image = 0
+    this.setImage(image)
+    const that = this
+    this.loop = setInterval(function () {
+      image = (image + 1) % imageCount
+      that.setImage(image)
+    }, 10000)
```

Comment:
```suggestion
    }, 17000)
```

Timing too short to explain any thing, we should test that